lib/ansiblelint/__init__.py

```diff
@@ -22,123 +22,15 @@
 from collections import defaultdict
 import logging
 import os
-import re
 
+from ansiblelint.rules import AnsibleLintRule  # noqa
```

Comment:
Please avoid `noqa` unless there's a serious reason to do so.

Reply:
That is keeping the API compatibility with older versions. Any custom rules would import the base class from here, as that was also included inside the docs.

AnsibleLintRule is likely the only effectively public API for the linter, others stuff are likely to be considered private.

Reply:
I improved the exception, explaining it.